arch/arm64/boot/dts/broadcom/rp1.dtsi

```diff
@@ -789,6 +805,16 @@
 				       "gpio24", "gpio25";
 				bias-disable;
 			};
+			rp1_dpi_18bit_cpadhi_gpio1: rp1_dpi_18bit_cpadhi_gpio1 { /* Mode 6 with GPIO1(DEN) but not GPIO0 */
```

Comment:
dpi? in my audio driver? (it's more likely than you think.)

Reply:
That one was a DO NOT MERGE. It possibly should have gone on a different branch...

Reply:
Gone now